doc/SAI-Proposal-Prefix-Compression.md

```diff
@@ -0,0 +1,271 @@
+#  Prefix Compression #
+-------------------------------------------------------------------------------
```

Comment:
Please consider merging both the documents. One doc explains how the meta-data is set, and the other doc explains how it is used.  Each doc is incomplete on its own.

Reply:
The reason I made two documents because Prefix Compression should be a standalone feature. In this case I am using the feature for ACL and added extensions in ACL.

The first document explains how Prefix Compression works as a feature. The feature allows adding entries of longest prefix and mapping those to a unique META value. 

The second document is an ACL specific document that explains how ACL would use that feature. In that document i do give a full example of creating a table/entry, and applying it to ACL

I want prefix compression not to be though of as an extension to ACL. It is a separate feature that ACL uses. Similar to other features that ACL uses like mirroring or sample packet. Sample packet stand alone is a feature but it can be used with ACL. Sample packet has its own documentation that is not associated with ACL.

Reply:
@rck-innovium, I updated the documentation. I combined both in a single document. Please let me know if this meets your request and resolve if it does